recipes/libx264/all/conanfile.py

```diff
@@ -113,16 +114,15 @@ def package(self):
             autotools.install()
         self.copy(pattern="COPYING", src=self._source_subfolder, dst='licenses')
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+        if self._is_msvc:
+            ext = ".dll.lib" if self.options.shared else ".lib"
+            tools.rename(os.path.join(self.package_folder, "lib", "libx264{}".format(ext)),
+                         os.path.join(self.package_folder, "lib", "x264.lib"))
```

Comment:
It would be nice to rename to `.dll.lib`.
Some (cmake) scripts depend on the static library suffix to differentiate between static and shared libraries.

Reply:
`.dll.lib` is not standard in Visual Studio, it can even break some build systems. CMake scripts relying on this suffix if compiler is Visual Studio are wrong.

(Extension of MinGW builds are not renamed here)

Reply:
I hope so. maybe some build systems (ffmpeg, I am looking at you) hard-code these library names and rename will break them, but I really hope most of libraries rely on pkg-config or cmake files to abstract over specific library names. but for things like ffmpeg patch might be needed.

Reply:
Moreover, the lib name depends on autotools version used. Since autoconf 2.71, it seems that Visual Studio libs generated with autotools don't have lib prefix and .dll.lib suffix anymore.

Reply:
Looks like vcpkg removes the `.dll` too.
So I'm good.